web/src/lib/components/forms/edit-user-form.svelte

```diff
@@ -34,6 +37,7 @@
           name,
           storageLabel: storageLabel || '',
           externalPath: externalPath || '',
+          quotaSizeInBytes: quotaSize ? convertToBytes(Number(quotaSize), 'GiB') : null,
```

Comment:
We do converFromBytes just to do convertToBytes again?

Reply:
The UI is in GB, so they type in 5.2 and then this concerts it to actual bytes which is what we store. On the other hand when we display it the original data is total bytes and we display it in GiB on the screen and in the input.